docusaurus/docs/e2e-test-a-plugin/get-started.md

```diff
@@ -124,6 +123,12 @@ export default defineConfig({
 });
 ```
 
+The authenticated state is stored on disk and the file name pattern is as follows: `<plugin-root>/playwright/.auth/<username>.json`. To prevent these files from being version controlled, you can add the following line to your `.gitignore` file.
```

Comment:
```suggestion
The authenticated state is stored on disk and the file name pattern is as follows: `<plugin-root>/playwright/.auth/<username>.json`. To prevent these files from being version controlled, you can add the following line to your `.gitignore` file:
```